torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6044,7 +6044,7 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
     UnaryUfuncInfo('abs',
                    aliases=('absolute', ),
                    ref=np.abs,
-                   dtypes=all_types_and_complex_and(torch.half, torch.bfloat16),
+                   dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
```

Comment:
cc @bertmaher since it looks like you touched the test (test_unary_ops in test_jit_fuser_te.py) recently. It seems like either NNC or TorchScript (or maybe both?) doesn't support torch.bool dtype. Could you let us know if that assessment is correct please?

As a workaround... does adding a "if dtype is torch.bool, continue" to https://github.com/pytorch/pytorch/blob/69da4b43812e48def999c04a3b1157b6504ef967/test/test_jit_fuser_te.py#L1277 work? 

(Here's the logs to the failing test, if that helps: https://app.circleci.com/pipelines/github/pytorch/pytorch/385228/workflows/a3986c59-6198-4e64-a74b-0e26304b748c/jobs/16199607 . Thanks in advance!)